### ISSUE TITLE
IOS : Phone Verification Not working , any special configuration 

### ISSUE THREAD

**Issue by yashhema** [rafaelsetragni/awesome_notifications#736]
Hello,
We have included awesome notification and we are not using fcm messaging. 

Not clear from documentation  ---

A: Is there anything besides configuration for awesome_notifications_fcm we need to do.
B : We are getting issues with Phone verfication (something like what is talked about in https://github.com/firebase/flutterfire/issues/9010), have you faced any issue like this

**Comment by rafaelsetragni** [comment]
Sorry, but I don't understand your problem. Can you describe it in more detail? What is "not working" and what is the expected behavior?
Also, I've never used firebase auth phone verification.

**Comment by yashhema** [comment]
Hello,
In our app we have enabled firebase phone verification . For android it works fine.
But in IOS we get error - https://github.com/firebase/flutterfire/issues/9010
So wanted to know , do we need to something special for firebase phone verification (some special thing)
regards

**Comment by rafaelsetragni** [comment]
Yes, phone verification on iOS does not work the same as on Android. On iOS, it uses silent push notifications to validate the phone before trying to use reCaptcha, like on Android. Since notifications are a global resource, different plugins trying to access it will end up conflicting and there's nothing what we can do about it.

To solve it I created a request to the firebase team to add an option to set firebase_auth to always use reCaptcha on iOS and you can check the progress here:
https://github.com/firebase/flutterfire/discussions/10875
